cypress/support/page_objects/registerPage.js

```diff
@@ -0,0 +1,16 @@
+
+class registerPage {
+    
+    signUpPage(first_name,last_name,mobile_phone,email,password) {
+        cy.get('.row > :nth-child(1) > .form-group > .pure-material-textfield-outlined > span').type(first_name)
```

Comment:
In this page you have simpler selectors, you could use the attribute "name" of the element.
Using simpler selectors you make your code more readable.